src/projection.js

```diff
@@ -98,11 +98,11 @@ export function Projection(
 }
 
 export function hasProjection({projection} = {}) {
-  if (projection == null) return false;
+  if (isIdentityProjection(projection)) return false;
   if (typeof projection.stream === "function") return true; // d3 projection
   if (isObject(projection)) ({type: projection} = projection);
   if (typeof projection !== "function") projection = namedProjection(projection);
-  return projection != null;
+  return !isIdentityProjection(projection);
```

Comment:
This doesn’t work because here `projection` will be `identity` (the symbol) rather than `"identity"` (the string). You can test with `projection: {type: "identity"}`.